openmdao/core/system.py

```diff
@@ -3511,7 +3511,7 @@ def get_io_metadata(self, iotypes=('input', 'output'), metadata_keys=None,
         return result
 
     def list_inputs(self,
-                    values=True,
+                    vals=True,
```

Comment:
I think this should be singular `val` to match `shape`, as well as the use of `val` everywhere else in the code.